ui/smite_priest/sim.ts

```diff
@@ -0,0 +1,241 @@
+				Stat.StatIntellect,
+				Stat.StatSpirit,
+				Stat.StatSpellPower,
+				Stat.StatShadowSpellPower,
+				Stat.StatHolySpellPower,
+				Stat.StatSpellHit,
+				Stat.StatSpellCrit,
+				Stat.StatSpellHaste,
+				Stat.StatMP5,
+			],
+			modifyDisplayStats: (player: Player<Spec.SpecSmitePriest>, stats: Stats) => {
+				return stats.withStat(Stat.StatSpellHit,
+						stats.getStat(Stat.StatSpellHit)
+						+ player.getTalents().shadowFocus * 2 * Mechanics.SPELL_HIT_RATING_PER_HIT_CHANCE
+						+ player.getTalents().focusedPower * 2 * Mechanics.SPELL_HIT_RATING_PER_HIT_CHANCE);
+			},
+
+			defaults: {
+				// Default equipped gear.
+				gear: Presets.P1_PRESET.gear,
+				// Default EP weights for sorting gear in the gear picker.
+				epWeights: Stats.fromMap({
+					[Stat.StatIntellect]: 1.38,
+					[Stat.StatSpirit]: 1.18,
+					[Stat.StatSpellPower]: 1,
+					[Stat.StatSpellHit]: 2.57,
+					[Stat.StatShadowSpellPower]: 0.05,
+					[Stat.StatHolySpellPower]: 0.95,
+					[Stat.StatSpellCrit]: 0.44,
+					[Stat.StatSpellHaste]: 0.28, // tricky because SP is tricky
+					[Stat.StatMP5]: 2.05,
+				}),
+				// Default consumes settings.
+				consumes: Presets.DefaultConsumes,
+				// Default rotation settings.
+				rotation: Presets.DefaultRotation,
+				// Default talents.
+				talents: Presets.StandardTalents.data,
+				// Default spec-specific settings.
+				specOptions: Presets.DefaultOptions,
+				// Default raid/party buffs settings.
+				raidBuffs: RaidBuffs.create({
+					arcaneBrilliance: true,
+					divineSpirit: TristateEffect.TristateEffectImproved,
+					giftOfTheWild: TristateEffect.TristateEffectImproved,
+				}),
+				partyBuffs: PartyBuffs.create({
+					drums: Drums.DrumsOfBattle,
+					bloodlust: 1,
+					manaSpringTotem: TristateEffect.TristateEffectRegular,
+					totemOfWrath: 1,
+					wrathOfAirTotem: TristateEffect.TristateEffectRegular,
+				}),
+				individualBuffs: IndividualBuffs.create({
+					blessingOfKings: true,
+					blessingOfWisdom: 2,
+				}),
+				debuffs: Debuffs.create({
+					judgementOfWisdom: true,
+					misery: true,
+					curseOfElements: TristateEffect.TristateEffectRegular,
+				}),
+			},
+
+			// IconInputs to include in the 'Self Buffs' section on the settings tab.
+			selfBuffInputs: [
+			],
+			// IconInputs to include in the 'Other Buffs' section on the settings tab.
+			raidBuffInputs: [
+				IconInputs.ArcaneBrilliance,
+				IconInputs.DivineSpirit,
+				IconInputs.GiftOfTheWild,
+			],
+			partyBuffInputs: [
+				IconInputs.SanctityAura,
+				IconInputs.MoonkinAura,
+				IconInputs.DrumsOfBattleBuff,
+				IconInputs.DrumsOfRestorationBuff,
+				IconInputs.Bloodlust,
+				IconInputs.WrathOfAirTotem,
+				IconInputs.TotemOfWrath,
+				IconInputs.ManaSpringTotem,
+				IconInputs.ManaTideTotem,
+				IconInputs.DraeneiRacialCaster,
+				IconInputs.EyeOfTheNight,
+				IconInputs.ChainOfTheTwilightOwl,
+				IconInputs.JadePendantOfBlasting,
+				IconInputs.AtieshWarlock,
+				IconInputs.AtieshMage,
```

Comment:
Doesn't have to be now, but you'll want to add Sanctity Aura here.

Reply:
I have it at the top of partyBuffInputs - will move to the bottom of the list. Is there a specific reason for ordering?

Reply:
Oh sorry, didn't notice that. This is fine. FYI the order you use here will determine the order in which the buffs appear in the UI.